scenarios1/01_An_Orcish_Assault.cfg

```diff
@@ -77,6 +77,24 @@
         [modifications]
             {TRAIT_LOYAL OVERLAY=""}
             {TRAIT_STRONG}
+            [advancement]
```

Comment:
I was suggesting to make this a trait so that it would be visible.

Something like _perfectionist: needs more exp to advance but deals 50% more damage._

Reply:
Oh wow.  3228 XP (total) required to attain the first AMLA.  I better choose wisely.  And now that you took away my ability to peak, I may never know my legacy.  Nice.

Though I do wish "aware of legacy possessed" would tell you what it was after you've taken it (as in in Unit Info).

Reply:
> Though I do wish "aware of legacy possessed" would tell you what it was after you've taken it (as in in Unit Info).

If you got something inconvenient there, you might simply change the last advancement to get something else than legacy revelation. Or pick a different advancement if you lose the scenario and start again. If it's actually shown as late as after the following AMLA, it's not so trivial to go back.